Source/WebCore/testing/Internals.mm

```diff
@@ -215,27 +212,6 @@ - (NSAttributedString *)_attributedStringForRange:(NSRange)range
     return !sandbox_check(pid, "iokit-open", static_cast<enum sandbox_filter_type>(SANDBOX_FILTER_IOKIT_CONNECTION | SANDBOX_CHECK_NO_REPORT), ioKitClass.utf8().data());
 }
 
-#if ENABLE(WEBGL) && PLATFORM(COCOA)
-bool Internals::platformSupportsMetal(bool isWebGL2)
```

Comment:
Also need to remove `Internals::platformSupportsMetal` from `testing/Internals.cpp`